_extras/guide.md

```diff
@@ -285,6 +285,10 @@ what, arise naturally once we start talking about using public services like
 GitHub to store files. Also, the discussion gives learners a chance to catch
 their breath after what is often a frustrating couple of hours.
 
+The Creative Commons family of licenses is recommended for many types of
+works but not software. Creative Commons [recommends][cc-faq-software]
```

Comment:
perhaps add something like "many types of works (including project documentation) but not software", so that learners can understand the context of learning about CC licenses in a software workshop.

Reply:
Good idea.  I updated this in 0e2fdee6d24ddaaae3b5fbb233ce233fa731f5b4.